Cargo.toml

```diff
@@ -117,7 +117,7 @@ version_check = "0.9.5"
 watchman_client = { version = "0.9.0" }
 whoami = "1.6.0"
 winreg = "0.52"
-
+winapi= {version = "0.3.9", features = ["winuser"] }
```

Comment:
nit: keep the blank line (i don't care strongly about the blank line per se, but someone probably put it here on purpose)